Rakefile

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env rake
+
+require 'rake/testtask'
+require 'rubocop/rake_task'
+
+# Rubocop
+desc 'Run Rubocop lint checks'
+task :rubocop do
+  RuboCop::RakeTask.new
+end
+
+# lint the project
+desc 'Run robocop linter'
+task lint: [:rubocop]
+
+# run tests
+task default: [:lint, 'test:check']
+
+namespace :test do
+  # run inspec check to verify that the profile is properly configured
+  task :check do
+    dir = File.join(File.dirname(__FILE__))
+    sh("bundle exec inspec check #{dir}")
+  end
+end
+
+# Automatically generate a changelog for this project. Only loaded if
+# the necessary gem is installed.
+# use `rake changelog to=1.2.0`
+begin
+  v = ENV['to']
+  require 'github_changelog_generator/task'
+  GitHubChangelogGenerator::RakeTask.new :changelog do |config|
+    config.future_release = v
```

Comment:
@chris-rock why? This both things are used by the log generator to get the information about the issues/PRs when accessing the project via GitHub API. Issues/PRs are always in the base repo within dev-sec organization.

If you do not want this things, just ignore it. The point is, if somebody works on the forks and not in the base repo within dev-sec org (like myself) and wants to update the Changelog - it does not work. You have either to change the remote `origin` to the base repo within dev-sec organization or set this options